client/components/spinner/docs/example.jsx

```diff
@@ -18,9 +18,22 @@ module.exports = React.createClass( {
 		return (
 			<div>
 				<p>
-					<strong>Please exercise caution in deciding to use a spinner in your component.</strong> A lone spinner is a poor user-experience and conveys little context to what the user should expect from the page. Refer to <a href="/devdocs/docs/reactivity.md">the <em>Reactivity and Loading States</em> guide</a> for more information on building fast interfaces and making the most of data already available to use.
+					<strong>Please exercise caution in deciding to use a spinner in your component.</strong>&nbsp;
+					A lone spinner is a poor user-experience and conveys little context to what the user should expect from the page.&nbsp;
+					Refer to <a href="/devdocs/docs/reactivity.md">the <em>Reactivity and Loading States</em> guide</a>&nbsp;
+					for more information on building fast interfaces and making the most of data already available to use.
 				</p>
+
+				<strong>Spinner</strong>
 				<Spinner />
+
+				<br />
+				<strong>counterClockwise: true</strong>
+				<Spinner counterClockwise={ true } />
+
+				<br />
+				<strong>delay: 5000</strong>
+				<Spinner delay={ 5000 } />
```

Comment:
It is not easily visible on the devdocs page how adding `delay` affects `Spinner` because until you get to Spinner, the delay is over and spinner is spinning as any other.

I think we should either add a text saying: "reload page to see the delay" or maybe increase the delay to `15000`?